Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/how_can_i_be_removed_from_a_team.md

---
title: How can I be removed from a team?
toc_hide: true
type: docs
tags:
  - administrator
  - team management
---

> To be removed from a team, you will need to reach out to a team admin and ask them to delete you from the Users tab.

Reviewer: ```suggestion
A team admin can [remove you from a team]({{< relref "/guides/models/app/settings-page/teams.md" >}})  from the **Users** tab of the team settings.
```
